src/Simulate.js

```diff
@@ -0,0 +1,16 @@
+import { MissionUtils } from '@woowacourse/mission-utils';
+const Random = MissionUtils.Random;

```

Comment:
```suggestion
import { Random } from '@woowacourse/mission-utils';
```
여기도 이렇게 import문을 줄여볼 수 있을 것 같아요🙃